src/main/webapp/app/pages/newsPage/NewsPageContent.tsx

```diff
@@ -634,7 +644,7 @@ export const NEWS_BY_DATE: { [date: string]: NewsData } = {
     updatedImplication: [
       [
         '3A',
-        'KMT2A',
```

Comment:
The same goes to updatedImplication which uses `UpdatedTxImplListItem` to render.